v2/src/components/recipeBoxes/guides.json

```diff
@@ -0,0 +1,79 @@
+{
+  "auth_modes": [
```

Comment:
For long recipe names the spacing is off

<img width="304" alt="Screenshot 2023-03-15 at 12 54 11 PM" src="https://user-images.githubusercontent.com/18233774/225236432-50882591-fcd5-483d-988d-221d192be474.png">